src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/ov_plugin/properties.cpp

```diff
@@ -24,7 +24,7 @@ INSTANTIATE_TEST_SUITE_P(ov_plugin_mandatory, OVPropertiesIncorrectTests,
 
 const std::vector<ov::AnyMap> default_properties = {
         {},
-        {ov::enable_profiling(true)},
+        {ov::enable_profiling(sw_plugin_in_target_device(ov::test::utils::target_device) ? false : true)},
```

Comment:
I think HW Plugin should have default value false as well.

Reply:
Correct. Agreed. Fixed